remoting/WFLY-19992-Enable-configuration-of-selected-XNIO-Remoting-parameters-for-remoting-objects-using-in-both-inbound-and-outbound-communication.adoc

```diff
@@ -0,0 +1,129 @@
+---
+categories:
+- core
+- remoting
+stability-level:
+default
```

Comment:
Is there a reason that you feel this needs to come it at the default stability level from the outset?

If we can it may be advisable to come it as Preview or Community and promote to default in a second step.

Reply:
Hmm, to be honest I wasn't sure whether the new levels would be applicable so I choose the default one. The one reason may be that this is issue has already causes problems to the customer and it would be good to provide it as stable feature pretty soon.